### DISCUSSION TITLE
Flatpak installation corruption

### DISCUSSION THREAD

**Discussion by aberba** [elementary/appcenter#1596] [Q&A] [answered]
Installation of eddy got stuck during installation so I canceled it. This I believe has made unable to install any other app under `appcenter` but I can still install under `appcenter(system-wide)`

![Screenshot from 2021-08-13 19 06 47](https://user-images.githubusercontent.com/8314648/129408458-2846d082-1bfb-4dda-8c2b-d50d58d1fb06.png)

**Comment by cassidyjames** [comment, marked as answer]
Can you try opening a Terminal and running:

```shell
flatpak --user repair
```

and try again? If that works, then this would resolve your issue if implemented: https://github.com/elementary/appcenter/issues/1523
